packages/terra-collapsible-menu-view/tests/jest/CollapsibleMenuViewHyperlink.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
+import CollapsibleMenuViewHyperlink from '../../src/CollapsibleMenuViewHyperlink';
+
+describe('CollapsibleMenuViewHyperlink', () => {
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallowWithIntl(<CollapsibleMenuViewHyperlink text="Testing" href="https://www.oracle.com/" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  // Props
+  it('should merge custom props', () => {
+    const wrapper = shallowWithIntl(<CollapsibleMenuViewHyperlink text="Testing" href="https://www.oracle.com/" className="Testing" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  describe('Collapsible Menu Context', () => {
+    it('should render a menu item', () => {
+      const context = { isCollapsibleMenuItem: true };
+      const wrapper = shallowWithIntl(<CollapsibleMenuViewHyperlink text="Testing" href="https://www.oracle.com/" />, { context }).dive();
+      expect(wrapper).toMatchSnapshot();
```

Comment:
we should assert for the existence of a menu item.

Reply:
The wrapper _is_ the menu item, so the snapshot will cover this

Reply:
Also for whatever reason, I can't seem to do wrapper.exists(Menu.Item) here, it just can't find it even though it is clearly rendering it in the snapshot :/

Reply:
> The wrapper _is_ the menu item, so the snapshot will cover this
Ah okay, I thought it was a nested element. That should be fine then.